### PR TITLE
Update Set-CsPhoneNumberAssignment.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Comment by RuiPereiraTabares** [comment]
hence currently i have a bug open for changing the exception, right now assiging a phone or setting enterprise voice enable to a user that his location is MO, we will say that the user does not has a license assigned, the error message is planned to be changed.

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ To remove a phone number from a user or resource account, use the [Remove-CsPhon
 
 If the cmdlet executes successfully, no result object will be returned. If the cmdlet fails for any reason, a result object will be returned that contains a Code string parameter
 and a Message string parameter with additional details of the failure.
-
+**Note**: Macao region is currently not supported for phone number assigment or enterprisevoice.
```
Comment (dariomws) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:40]:
```suggestion

**Note**: Macao region is currently not supported for phone number assignment or Enterprise Voice.

```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by tiburd** [comment]
@RuiPereiraTabares 
Hi, Rui,
Because you are a Microsoft employee, please be sure to [link your Microsoft and GitHub accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github?branch=main#link-your-github-and-microsoft-accounts).
Thanks!

**Comment by RuiPereiraTabares** [comment]
@tiburd I have updated my account as a microsoft github account can you verify?

**Comment by tiburd** [comment]
> @tiburd I have updated my account as a microsoft github account can you verify?
@RuiPereiraTabares 
Thank you!
It's now listed as being connected:
image

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by tseward** [comment]
@RuiPereiraTabares is this a permanent (or very long term) issue or temporary? If temporary, I would suggest this would fit better as a Service Health issue in the admin dashboard on admin.microsoft.com. If it is permanent or very long term, then I can see why adding this to the documentation would be beneficial.

**Comment by RuiPereiraTabares** [comment]
@tseward is going to be pretty long term (2+yers), we will change our exception to this region is not supported. Therefore is better to have it somewhere documented.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@RuiPereiraTabares it seems Microsoft uses "Macau" in other articles like https://docs.microsoft.com/en-us/microsoftteams/what-is-phone-system-in-office-365 @JohanFreelancer9 ?

also, what about this? is it outdated? 

> Phone System is available world wide except for Macau.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ To remove a phone number from a user or resource account, use the [Remove-CsPhon
 If the cmdlet executes successfully, no result object will be returned. If the cmdlet fails for any reason, a result object will be returned that contains a Code string parameter
 and a Message string parameter with additional details of the failure.
 
+**Note**: Macao region is currently not supported for phone number assignment or Enterprise Voice.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:41]:
```suggestion
**Note**: Macau region is currently not supported for phone number assignment or Enterprise Voice.

```

**Comment by JohanFreelancer9** [comment]
@get-itips, thanks for pointing out this discrepancy. Although "Macao" is the official name, "Macau" also seems to be widely used, certainly in a number of Microsoft docs. So for the sake of consistency, I would suggest we stick to "Macau". @RuiPereiraTabares, could you please commit the suggestion. Thanks.

**Comment by RuiPereiraTabares** [comment]
done @JohanFreelancer9

**Comment by RuiPereiraTabares** [comment]
> @RuiPereiraTabares it seems Microsoft uses "Macau" in other articles like https://docs.microsoft.com/en-us/microsoftteams/what-is-phone-system-in-office-365 @JohanFreelancer9 ?
> 
> also, what about this? is it outdated?
> 
> > Phone System is available world wide except for Macau.

On regards this, @get-itips  i discussed internally to at least have one article where we mention Macao/Macau, if you use the blame tool you can see that over that article Macau was added 3 days ago, our internal discussion started last friday so mostlkely the PM pushed for this change.  I think also having this documented on this article will be good

**Comment by tiburd** [comment]
Sent email to PMs to ask whether this information is better suited for a Service Health Issue.

**Pull request merged**